### DISCUSSION TITLE
Welcome to afrog Discussions!

### DISCUSSION THREAD

**Discussion by zan8in** [zan8in/afrog#54] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bigyear** [comment]
报告html内容乱码，如何解决

**Reply by zan8in** [reply]
部分 tcp PoC 报告显示乱码是由于返回结果为二进制结果（ASCII码），并非 utf8 与 gbk 乱码。
